### PR TITLE
[Svace] Fix tizen svace issue

### PR DESCRIPTION
**Pull request by DonghakPark**
## In this PR

 **Fix tizen svace issue** 

Fix Svace issues on tensor.h, blas_neon.cpp, layer_context.h

""""""""""""""""""""""""""""""""""""""""""""""
**Self evaluation:**
Build test: [x]Passed [ ]Failed [ ]Skipped
Run test: [x]Passed [ ]Failed [ ]Skipped_
""""""""""""""""""""""""""""""""""""""""""""""

Signed-off-by: Donghak PARK <donghak.park@samsung.com>

### CODE REVIEW

**Review by djeong20 — APPROVED** [review]
LGTM

**Comment by SeoHyungjun** [comment]
LGTM

File: nntrainer/tensor/blas_neon.cpp [hunk 1]
```diff
@@ -248,7 +248,7 @@ void sgemv_transpose_neon(const float *A, const float *X, float *Y,
     unsigned int n = cols / 4;
     bool *initialized = (bool *)malloc(sizeof(bool) * n);
     if (initialized == nullptr) {
-      ml_loge("failed to malloc");
+      ml_loge("Error : Memory allocation failed");
```
Comment (myungjoo) [inline nntrainer/tensor/blas_neon.cpp:251]: initialized is not transfered to its caller.

Consider using ```auto initialized = make_unique<bool []>(n);```. You won't need nullptr check, either, then.

Comment (DonghakPark) [reply nntrainer/tensor/blas_neon.cpp:251]: Thank you for review, i will fix and update

**Review by myungjoo — APPROVED** [review]

**Pull request merged**